### ISSUE TITLE
Feature Request - RDAirPlay

### ISSUE THREAD

**Issue by electricurrent** [ElvishArtisan/rivendell#358]
Just an idea, I personally think the color of the "Next Timed Start" box in the top left of RDAirPlay should be Green if there is a surplus in the hour because that's usually what you shoot for anyway, and the current Red looks misleading..

**Comment by ElvishArtisan** [comment]
Correct. That choice of colors was heavily influenced by the setting in which Rivendell was developed in its early years, which was dominated by live satellite-delivered programming.

I'm not averse to making the colors configurable, but I'm not sure that 'satellite' vs. 'live-assist' are particularly meaningful terms. What are some others?

**Comment by deltecent** [comment]
I like the idea of making the colors configurable.

**Comment by pabsungenis** [comment]
Perhaps something along the lines of “Backtiming: Hard/Flexible?”
    	
    	

    	Get Outlook for iOS
    
  




On Fri, Jan 25, 2019 at 12:34 PM -0500, "Deltec Enterprises" <notifications@github.com> wrote:












I like the idea of making the colors configurable.

**Comment by ElvishArtisan** [comment]
How about:
Backtiming Colors: 'Less is better' | 'More is better'

**Comment by pabsungenis** [comment]
I was thinking of fewer words but your “less is better” suggestion is probably the best. Descriptive and accurate. 
    	
    	

    	Get Outlook for iOS
    
  




On Fri, Jan 25, 2019 at 12:45 PM -0500, "Fred Gleason" <notifications@github.com> wrote:












How about:

Backtiming Colors: 'Less is better' | 'More is better'

**Comment by validmirror** [comment]
i hate the idea of changing this...pretty colors and alterations  = getting away from the simple, easy to understand interface that we have now, which I believe is one of the plus factors 
of the program...
